### PR TITLE
use @types

### PR DESCRIPTION
**Pull request by Kaffiend**
havent really worked with generators, but from what i can decypher at a glance something like this i what i was meaning.

### CODE REVIEW

**Comment by Kaffiend** [comment]
i think thats it, first time working on a generator. 😄

**Comment by buehler** [comment]
Cool :-)

Would you mind to add a corresponding line to the changelog?
like
```md
#### Changed
- ...
```

File: app/index.ts [hunk 1]
```diff
@@ -102,18 +102,29 @@ class GiuseppeGenerator extends yeomanGenerator.Base {
...
+                'tslint',
+                '@types/express',
+                '@types/express-serve-static-core',
+                '@types/http-status',
+                '@types/mime',
+                '@types/node',
+                '@types/serve-static',
+                '@types/body-parser'
+                ],
             devTypings = [],
```
Comment (buehler) [inline app/index.ts]: Those should be unused variables now right? (`devTypings` and `devGlobalTypings`)

Comment (buehler) [reply app/index.ts]: Or more typings in general :-)

File: app/templates/tsconfig.json [hunk 2]
```diff
@@ -9,6 +9,10 @@
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true
   },
+  "typeRoots": [
```
Comment (buehler) [inline app/templates/tsconfig.json]: isn't this default by typescript? As I'm aware of, @types is the default folder to look for typings, do you really need to provide the `typeRoots` info?

**Review by buehler — CHANGES_REQUESTED** [review]
General good idea, I just have some small comments.

**Comment by Kaffiend** [comment]
I believe your right, i usually have it out of explicitness or if im doing mixed typings when there isnt any @types for a lib yet. Yes those should be unused variables now so they can be dropped out unless we encounter an issue with a typing and need to override with a legacy typing file

**Comment by mfeltscher** [comment]
LGTM, @buehler WDYT? :shipit:

**Review by buehler — APPROVED** [review]

**Comment by buehler** [comment]
Yep, :shipit: 
Thank you @Kaffiend

**Pull request merged**